app/dashboards/article_dashboard.rb

```diff
@@ -10,8 +10,7 @@ class ArticleDashboard < Administrate::BaseDashboard
   ATTRIBUTE_TYPES = {
     user: Field::BelongsTo,
     user_id: UserIdField,
-    second_user_id: Field::Number,
-    third_user_id: Field::Number,
+    co_author_ids: Field::Number,
```

Comment:
should this be `Field::Select` ?

Reply:
I actually have no idea. I think `Field::Number` makes sense given the description for `ATTRIBUTE_TYPES`, but again, I'm not sure. Why do you think that this field should possibly be `Field::Select` instead @rhymes? 🤔

Reply:
Because I think `::Number` is for single values, `::Select` is to select multiple values. I haven't tested though, maybe see in `localhost:3000/resource_admin` which is the old admin :D

Reply:
I'm still trying to figure this out.. after poking around at this for a _while_ yesterday, I found a solution (using `Field::Select` with the `:collection` option that defaults to an empty array) that should work, but I was unable to get it to work properly. I'm circling back around to this now that I've made other changes and hopefully with a fresh set of eyes, I will be able to make it work @rhymes ! 😄